packages/ardour-git/ardour-git.pacscript

```diff
@@ -44,14 +46,46 @@ build() {
 install() {
 	sudo ./waf i18n --destdir="$STOWDIR/$name"
 	sudo ./waf install --destdir="$STOWDIR/$name"
-	cd "$STOWDIR/$name/usr/bin"
 	
-	sudo rm ardour6-copy-mixer ; sudo ln -s "../lib/ardour6/utils/ardour-util.sh" "/usr/bin/ardour6-copy-mixer"
-	sudo rm ardour6-export ; sudo ln -s "../lib/ardour6/utils/ardour-util.sh" "/usr/bin/ardour6-export"
-	sudo rm ardour6-fix_bbtppq ; sudo ln -s "../lib/ardour6/utils/ardour-util.sh" "/usr/bin/ardour6-fix_bbtppq"
-	sudo rm ardour6-new_empty_session ; sudo ln -s "../lib/ardour6/utils/ardour-util.sh" "/usr/bin/ardour6-new_empty_session"
-	sudo rm ardour6-new_session ;  sudo ln -s "../lib/ardour6/utils/ardour-util.sh" "/usr/bin/ardour6-copy-new_session"
+	wait
```

Comment:
What is this `wait` for?

Reply:
the waf installation sometimes gets caught in a loop. I thought that it may have been caused by the rm's